frontend/src/app/search/page.tsx

```diff
@@ -10,7 +10,7 @@ import SearchCallToAction from "../../components/search/SearchCallToAction";
 import { SearchForm } from "./SearchForm";
 import { convertSearchParamsToProperTypes } from "../../utils/convertSearchParamsToStrings";
 import { cookies } from "next/headers";
-import { getSearchFetcher } from "../../services/searchfetcher/SearchFetcherUtil";
+import { getSearchFetcher } from "../../services/search/searchfetcher/SearchFetcherUtil";
```

Comment:
I also moved the `SearchFetcher` classes from `/services/searchfetcher` to `services/search/searchfetcher/` so there's a bunch of updates in this PR for that